src/main.rs

```diff
@@ -206,7 +206,7 @@ fn git_diff(path: &Path) -> Result<String, clap::error::Error> {
 // Takes the `String` output of `git_diff` above, and filters out irrelevant
```

Comment:
Can you update `String` to be `&str` in the comment here?

Reply:
ah, yes! thanks for catching it 🙇

Reply:
I've directly squashed the fix into 2cf8939. 🙏